sda-commons-server-spring-data-mongo/src/test/java/org/sdase/commons/server/spring/data/mongo/MongoConfigurationTest.java

```diff
@@ -80,4 +80,32 @@ void shouldNotBeValidIfOnlyDatabaseWasSet() {
     MongoConfiguration config = new MongoConfiguration().setDatabase("records");
     assertThat(config.isValid()).isFalse();
   }
+
```

Comment:
We could also add a test without `ssl` or `tls` in the connection string to ensure that `isUseSsl` is `false`.

Reply:
That is tested [here](https://github.com/SDA-SE/sda-dropwizard-commons/pull/2288#discussion_r1166287322), isn't it?.